soap/src/java/com/zimbra/soap/account/message/AuthResponse.java

```diff
@@ -280,6 +283,15 @@ public String getTrustedToken() {
     public ZmBoolean getTrustedDevicesEnabled() { return trustedDevicesEnabled; }
     public AuthResponse setTrustedDevicesEnabled(boolean bool) { this.trustedDevicesEnabled = ZmBoolean.fromBool(bool); return this; }
 
+    @GraphQLQuery(name="resetPassword", description="if true then auth token will be used to change password")
+    public String getResetPassword() {
+        return resetPassword;
+    }
+
+    public void setResetPassword(String resetPassword) {
+        this.resetPassword = resetPassword;
+    }
```

Comment:
1. Can you change it to `boolean`, in the same way as `trustedDevicesEnabled`
```
    @GraphQLQuery(name="trustedDevicesEnabled", description="Denotes if trusted devices are enabled")
    public ZmBoolean getTrustedDevicesEnabled() { return trustedDevicesEnabled; }
    public AuthResponse setTrustedDevicesEnabled(boolean bool) { this.trustedDevicesEnabled = ZmBoolean.fromBool(bool); return this; }
```

2. the changes need to be added to admin namespace as well if my understanding is correct.
`soap/src/java/com/zimbra/soap/admin/message/AuthResponse.java`

Reply:
I'm not sure, but regarding `setResetPassword`, `return this;` may not be necessary.